src/components/AppliedFiltersDisplay.tsx

```diff
@@ -109,6 +112,18 @@ export function AppliedFiltersDisplay(props: AppliedFiltersDisplayProps): JSX.El
 
   const handleRemoveStaticFilterOption = (filter: DisplayableFilter) => {
```

Comment:
what happens if the filter is removed by clicking on the checkbox option for the static filter or the facet? should the duplicate filter also be removed in that case?

Reply:
Right now we're assuming that static filters have higher priority over facets in AppliedFilters when we dedupe. I'm not super happy making this assumption but looking into it